tests/backup/backup_upgrade_test.go

```diff
@@ -711,6 +766,49 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", Label(TestCase
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying restoration [%s] of next all namespaces schedule backup [%s] in cluster [%s]", restoreName, nextScheduleBackupName, restoreName))
 			restoreNames = append(restoreNames, restoreName)
 		})
+		if IsKubevirtInstalled() {
```

Comment:
and PxBackup Version >= 2.7.0 just to be safe?

Reply:
We will not be running this test on pre 2.7.0 versions

Reply:
No upgrade hops? 2.5.0, 2.6.0, and 2.7.0?

Reply:
@mkoppal-px @kphalgun-px is correct we need to check if we are currently running 2.6.0 then only then only run the setups to create the vm deployment.

Reply:
@kshithijiyer-px We can deploy VMs in 2.4.0 setup as well right?

Reply:
@kphalgun-px We don't do hops AFAIK

Reply:
Perhaps we should also use hops here as well, like what PXBackupClusterUpgradeTest does. We can deploy VMs in version 2.4.0 given Kubevirt is installed. However, when this test case is run on such a setup—for instance, upgrading from 2.4.0 to 2.5.0—it attempts to back up the VM as well, and it will fail.

Reply:
Px-Backup customers usually do not do hop upgrades. If they are on 2.4.0 we will ask them to upgrade to 2.7.0

Reply:
Make sense.

Reply:
This is how I am implementing 
```
func IsCurrentPxBackupVersionGreaterThanOrEqualTo(v string) (bool, error) {
	currentBackupVersionString, err := GetPxBackupVersionSemVer()
	if err != nil {
		return false, err
	}
	currentBackupVersion, err := version.NewSemver(currentBackupVersionString)
	if err != nil {
		return false, err
	}
	versionSemVer, err := version.NewSemver(v)
	if err != nil {
		return false, err
	}
	return currentBackupVersion.GreaterThanOrEqual(versionSemVer), nil
}
```

```
checkPxbVersionForKubevirt, err := IsCurrentPxBackupVersionGreaterThanOrEqualTo("2.7.0")
		log.FailOnError(err, "Checking if current px-backup version is greater than or equal to 2.7.0")
		if IsKubevirtInstalled() && checkPxbVersionForKubevirt {
			Step("Taking backup of kubevirt VM", func() {
```

Reply:
@kshithijiyer-px Please check

Reply:
This is nice looks good to me. Thanks for closing it up @mkoppal-px